docs/getting-started/README.md

```diff
@@ -0,0 +1,121 @@
+# Getting Started
```

Comment:
Was it an accident that you added this file back? My PR moved it from this subdirectory to the docs folder named getting-started.md.